kratos/tests/cpp_tests/strategies/builder_and_solvers/test_builder_and_solver.cpp

```diff
@@ -571,141 +536,6 @@ namespace Kratos
             KRATOS_CHECK_LESS_EQUAL(std::abs((rA(21,21) - 185056985.8178827166557312)/rA(21,21)), tolerance);
         }
 
-        /**
-         * Checks if the block builder and solver with constraints performs correctly the assemble of the system
-         */
-        KRATOS_TEST_CASE_IN_SUITE(ExtendedDisplacementBlockBuilderAndSolverWithConstraints, KratosCoreFastSuite)
```

Comment:
I would comment this until is fixed, the test takes some time to create...

Reply:
I though this is a duplicate

It used the B&S that is removed in this PR

I can try to use the baseclass

Reply:
I brought back the tests and made a comment

I checked, the results are not the same when I use the baseclass (which was expected), but I don't feel confident updating them